containers/hbadeals/rsem2hbadeals.R

```diff
@@ -152,25 +158,48 @@ all      <- colnames(countsData)
 toRemove <- c("transcript_id","gene_id")
 toKeep   <- all [!all  %in% toRemove]
 
-# Apply exclusion criterion 1: Filter out transcripts with cumsum across samples lower than the N, cohort size
-# message("\nApplying exclusion criterion 1:  ( discard transcripts with very low count across samples) ..")
-countsData <- countsData[rowSums(countsData[,toKeep] > 0 ) >= sample_size, ]
```

Comment:
This should be replaced with @karleg 's suggestion:

```R
num.case=length(case)
num.control=length(control)
N_sample=min(num.case,num.control)
countsData=countsData[rowSums(countsData[,3:(2+num.control)]>0)>=N_sample*0.1,]
countsData=countsData[rowSums(countsData[,(2+num.control+1):ncol(countsData)]>0)>=N_sample*0.1,]
```

Reply:
It should be >N_sample*0.9 (strictly greater than 0.9 times the number of samples)

Reply:
Thanks for catching this one @karleg , https://github.com/TheJacksonLaboratory/coronavirus-nf/commit/5e9890b48ecf06b50316c1ae78876ff4eec385a2 fixes it.